cmd/skupper/README.md

```diff
@@ -0,0 +1,41 @@
+# Skupper CLI
+
+The [Skupper](https://skupper.io/) command line enables you to:
+
+* Create a Virtual Application Network (VAN) site
+* Connect to other VAN sites
+* Connect services
+
+## Getting `skupper`
+
+To install `skupper`, download the [latest release](https://github.com/skupperproject/skupper/releases).
+
+
+## Using `skupper`
+
+To create your first VAN site:
+
+```
+skupper init
+skupper connection-token /path/to/mysecret.yaml
+```
+
+To expose a service from the first site:
```

Comment:
Doesn't need to be in the first site. Could be in either or indeed both.

Reply:
I've edited to include this (and your other comment)